docs/docs/installation/configuration/Openshift.md

```diff
@@ -0,0 +1,23 @@
+---
+comments: true
+---
+
+# Installing on Openshift
```

Comment:
```suggestion
# Installing on OpenShift
```

Reply:
I don't think [OpenShift is an approved word, only Openshift.](https://github.com/keptn/lifecycle-toolkit/blob/c5d2b99d4501de5c59e49486a8f636566d88f1b7/.github/actions/spelling/expect.txt#L467)

Do you want me to change that?

Reply:
you are right! please ignore the suggestions